components/eventing-controller/config/crd/kustomization.yaml

```diff
@@ -5,7 +5,6 @@ resources:
 - bases/eventing.kyma-project.io_subscriptions.yaml
 - bases/eventing.kyma-project.io_eventingbackends.yaml
 - external/apirules-gateway-kyma-project-io.yaml
```

Comment:
Should this also be deleted? since you are deleting the `components/eventing-controller/config/crd/external/apirules-gateway-kyma-project-io.yaml`?

Reply:
Why should we delete the APIRules?